src/index.ejs

```diff
@@ -12,6 +12,16 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <% if(__TARGET__ === 'mobile') { %>
   <script src="cordova.js" defer></script>
+  <% } else { %>
+  {{.CozyClientJS}}
```

Comment:
@enguerran if I understand well, you package a standalone version of `cozy-client-js` in your mobile bundle, so it's ok to have the dependency injected in web version only, true?